server/src/main/scala/org/http4s/server/staticcontent/WebjarService.scala

```diff
@@ -12,7 +12,7 @@ object WebjarService {
 
   /** [[org.http4s.server.staticcontent.WebjarService]] configuration
     *
-    * @param blockingExecutionContext execution context for blocking I/O
+    * @param blocker execution context for blocking I/O
```

Comment:
Same as above.

Reply:
`Blocker` is just an execution context so I think the description is valid

Reply:
I agree with @aeons here.  A `Blocker` is a wrapper that exists to distinguish blocking execution contexts from CPU-bound execution contexts, and steers people toward better thread pool separation through types.

Reply:
Oops.  GitHub folded the diff, so I only saw the doc, and not the type.  I thought we were trying to use an `ExecutionContext` as a `Blocker`.  I still agree with @aeons that we should not have been quoting it as an `ExecutionContext`, but I think this phrasing is fine.

Reply:
Not 100% sure I understand what you mean by refer to `Blocker`. As `Blocker` is the type of the parameter does it need to also be in the description? Something like "`Blocker` for blocking I/O"?

Reply:
I think as is, "execution context for blocking I/O", is a good description.  

I would like us to avoid calling a blocker an `ExecutionContext`, with that capitalization, in backticks, which makes it look like a type name.